test/runtests.jl

```diff
@@ -78,29 +80,29 @@ datadir = joinpath(@__DIR__,"data")
         active[i,j,k] = false
       end
     end
-    reals = iqsim(TI, (10,10,10), size(TI), hard=shape)
+    reals = iqsim(TI, (10,10,10), size(TI), hard=shape, resource=testresource)
     @test all(isnan.(reals[1][.!active]))
     @test all(.!isnan.(reals[1][active]))
 
     # masked training image
     TI = ones(20,20,20)
     TI[:,5,:] .= NaN
-    reals = iqsim(TI, (10,10,10), size(TI))
+    reals = iqsim(TI, (10,10,10), size(TI), resource=testresource)
     @test reals[1] == fill(1.0, size(TI))
     TI[1,5,:] .= 0
-    reals = iqsim(TI, (10,10,10), size(TI))
+    reals = iqsim(TI, (10,10,10), size(TI), resource=testresource)
     @test reals[1] == fill(1.0, size(TI))
 
     # masked domain and masked training image
     TI = ones(20,20,20)
     TI[:,5,:] .= NaN
     aux = fill(1.0, size(TI))
     shape = Dict(CartesianIndex(i,j,k)=>NaN for i=1:20, j=5, k=1:20)
-    reals = iqsim(TI, (10,10,10), size(TI), hard=shape)
+    reals = iqsim(TI, (10,10,10), size(TI), hard=shape, resource=testresource)
```

Comment:
Same here, and in the rest of the tests...

Reply:
Removed.